src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas2/ApplicationNoteService.kt

```diff
@@ -26,6 +27,7 @@ class ApplicationNoteService(
   private val httpAuthService: HttpAuthService,
 ) {
 
+  @Transactional
```

Comment:
I'm not sure we need this to be wrapped in a transaction, as the only thing we're saving here is a `note` (rather than, say, a note and an application), but I could be misunderstanding...

Reply:
ah yeah that's a good point, I think I was thinking we should add it to any methods that do something to the database, but there's no point if it's just one transaction as if it fails there's nothing else to roll back?